bindings/rust/s2n-tls/src/testing/resumption.rs

```diff
@@ -63,13 +64,15 @@ mod tests {
     }
 
     #[test]
-    fn resume_session() -> Result<(), Box<dyn Error>> {
+    fn resume_tls12_session() -> Result<(), Box<dyn Error>> {
```

Comment:
There is a test named `resume_tls13_session` below and this one is tls1.2 specific.